src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -553,6 +567,13 @@ public static DynamoModel Start(IStartConfiguration configuration)
         /// <param name="config">Start configuration</param>
         protected DynamoModel(IStartConfiguration config)
         {
+            if (config is DefaultStartConfiguration)
+            {
+                // This is not exposed in IStartConfiguration to avoid a breaking change.
```

Comment:
do you want to add a TODO?

Reply:
The next line is a `TODO` without a prefix, so I'll add it there.

Reply:
Thanks for adding it @mmisol , can you mark it specially for 3.0? A bit easier to track all the 3.0 issues in code base

Reply:
Will do 👍